internal/gencli/completion_file_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gencli
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/googleapis/gapic-generator-go/internal/txtdiff"
+)
+
+func TestCompletionFile(t *testing.T) {
+	defer func() {
```

Comment:
`go test` should catch panics automatically right?

Reply:
Yes, but the resulting message is not useful in debugging the test. The panic I aim to catch here is from `tools/import` panicking do to a formatting error, typically a result of a broken generation.